Dockerfile

```diff
@@ -7,17 +7,18 @@ ENV PIP_NO_CACHE_DIR=1 \
     PYTHONDONTWRITEBYTECODE=1 \
     PYTHONUNBUFFERED=1
 
+WORKDIR /app
+
 # Only requirements to cache them in docker layer so we can skip package
 # installation if they haven't changed
 COPY requirements.txt .
-RUN pip install --requirement requirements.txt
+RUN pip install --no-cache-dir --requirement requirements.txt
```

Comment:
This is fine, it makes the a bit image smaller.

If you really want some fun, we can enable buildkit and get a persistent cache, which would speed up rebuilding the docker image locally. But probably not worth it.

Reply:
Definitely something to add later if we get anyone using the docker-compose flow in anger I think!